projects/module-2/whats-cookin-part-one.md

```diff
@@ -0,0 +1,278 @@
+* Fetch calls
+
+---
+
+### Extension Options
+* Implement one of the listed 3rd party libraries or one of your choosing with instructor approval.
+* Expand your filtering functionality to include multiple tags and be able to search by both name and ingredients.
+* Implement a rating system would be helpful for a user! This could include a 5 star rating system, a way to leave reviews, etc!
+* You may also collaborate with instructors to personalize an extension for this project.
+
+<section class="note">
+## 3rd Party Libraries
+
+You may choose **ONE** of the following 3rd party libraries to incorporate into your app *as an extension (aka not expected, not required)* if interested. Your group may want to do a research spike to see which library, if any, makes sense for your application. Please get instructor approval first before implementing any 3rd-party libraries (including these). 
+
+* [Micromodal](https://www.npmjs.com/package/micromodal){:target='blank'}
+* [GlideJS](https://www.npmjs.com/package/@glidejs/glide){:target='blank'}
+* [InteractJS](https://interactjs.io/){:target='blank'}
+
+</section>
+
+---
+
+### Minimum Collaboration and Professionalism Expectations
+* Team holds daily standups throughout project.
+* Commits are atomic and frequent, effectively documenting the evolution/progression of the application. There is no more than a 10% disparity in project contributions between teammates.
+* A project board is utilized (and updated throughout the project) with Github issues and labels.
+* Team uses branches, PRs and thorough code reviews to add new code to the main branch.
+* The README is formatted well and at a minimum contains:
+  * Overview of project and goals
+  * Overview of technologies used, challenges, wins, and any other reflections
+  * Screenshots/gifs of your app
+  * List of contributors
+* **Team collaborates effectively to accomplish the shared goal.  Team productively and professionally works through challenges and conflicts to ensure all team members are able to be heard and contribute throughout the project.**  
+  * Instructors are available to offer support and guidance but conversations around what *is* and what *is not* working are expected to be led by the team members themselves.
+
+---
+
+## Rubric
+
+For the rubric sections below, you will be scored as **Wow**, **Yes** or **Not Yet** depending on whether you have demonstrated competency in that area. Each section lists examples of what types of things we may be looking for as demonstrations of competency. Just as there are many ways to approach code, there are many many ways to demonstate competency.  There are just some examples. 
+
+### Functional Expectations
+* Wow: Application fulfills all requirements *as well as* an extension.
+* Yes: Application fulfills all requirements of iterations 1-3 without bugs.
+* Not Yet: Application crashes or has missing functionality or bugs.
+
+### Fundamental JavaScript & Style / OOP / Fetch
```

Comment:
I know this isn't up for review currently, but I was utilizing this as a guide for a later M2 project and I figured I'd throw a comment here saying that: 

- I love this so much. It's thoughtful and much better guides students than the original
- I love how you included and phrased the functional paradigm pieces. 
- The only thing I noted is that OOP is still included in this header on line 249

Reply:
Ah Cass you have such a good eye!  Thanks for catching that!